### PR TITLE
Updated urls

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2100

### CODE REVIEW

File: api/.env.default [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NETWORKS={"URL":"wss://rpc.v2.devnet.cere.network/ws","MNEMONICS":"","NETWORK":"DEVNET"},{"URL":"wss://rpc.v2.testnet.cere.network/ws","MNEMONICS":"","NETWORK":"TESTNET"},{"URL":"wss://rpc.v2.qanet.cere.network/ws","MNEMONICS":"","NETWORK":"QANET"}
+NETWORKS={"URL":"wss://archive.devnet.cere.network/ws","MNEMONICS":"","NETWORK":"DEVNET"},{"URL":"wss://archive.testnet.cere.network/ws","MNEMONICS":"","NETWORK":"TESTNET"},{"URL":"wss://archive.qanet.cere.network/ws","MNEMONICS":"","NETWORK":"QANET"}
```
Comment (AndreiNavoichyk) [inline api/.env.default]: Revert to rpc where it was rpc

Comment (DmitriyMolch) [reply api/.env.default]: fixed

File: backend/backend.config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ require('dotenv').config();
 
 module.exports = {
   substrateNetwork: process.env.SUBSTRATE_NETWORK || 'kusama',
-  wsProviderUrl: process.env.WS_PROVIDER_URL || 'wss://rpc.v2.devnet.cere.network/ws',
+  wsProviderUrl: process.env.WS_PROVIDER_URL || 'wss://archive.devnet.cere.network/ws',
```
Comment (AndreiNavoichyk) [inline backend/backend.config.js:6]: Revert to rpc

**Review by AndreiNavoichyk — APPROVED** [review]

File: backend/backend.config.js (hunk 2 above)
Comment (DmitriyMolch) [reply backend/backend.config.js:6]: I thought that for `cerestats` we should update all to archive. Fixed.

Comment (DmitriyMolch) [reply backend/backend.config.js:6]: If you look into [k8 configuration ](https://github.com/cere-io/infra-k8s-deployment/blob/286bee57ae18e24b30f49d00a6dd0a43be4d26a4/cerestats/cerestats-crawler/envs/dev.yaml#L9) then see  that it has  `archive`. So why `cerestats` should have different logic?

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**